src/components/BillCard/SmallBillCard/SmallBillCard.jsx

```diff
@@ -1,16 +1,39 @@
 import React from "react";
+import PropType from "prop-types";
+import {FaBus, FaCar, FaQuestion, FaShoppingBag, FaShoppingCart, FaUtensils } from "react-icons/fa";
+
+import { filterDateTime } from '../../../helpers/DateTime';
+
 import "./styles.scss";
 
-const SmallBillCard = ({ activeBill, bill, filterDateTime, billIcons }) => (
+const getBillIcons = (category) => {
```

Comment:
This should go in another file to be reused (as in a component to be reused rather than just in SmallBillCard)